src/kudu/tserver/tablet_service.h

```diff
@@ -0,0 +1,291 @@
+// Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
why are we bringing in this file. Most of the RPCs in this file don't make sense.

Reply:
I think this is got brought in in the backport (I developed this feature upstream so I'd have a nice testing framework) so I can clean this stuff up.